src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -81,11 +83,18 @@ public AssemblyScannerResults GetScannableAssemblies()
 
             var assemblies = new List<Assembly>();
 
-            foreach (var assemblyFile in ScanDirectoryForAssemblyFiles(baseDirectoryToScan, ScanNestedDirectories))
+            var directoriesToScan = string.IsNullOrWhiteSpace(AdditionalAssemblyScanningPath)
+                ? new[] {baseDirectoryToScan}
+                : new[] {baseDirectoryToScan, AdditionalAssemblyScanningPath};
+
+            foreach (var directoryToScan in directoriesToScan)
             {
-                if (TryLoadScannableAssembly(assemblyFile.FullName, results, out var assembly))
+                foreach (var assemblyFile in ScanDirectoryForAssemblyFiles(directoryToScan, ScanNestedDirectories))
```

Comment:
This means that nested directories for the additional scanning path will also be potentially scanned. I'm wondering if this should be an option for the additional scanning path.

Reply:
I don't know but assuming the assumptions about the primary path should apply to the additional path.